packages/translations/src/languages/de.ts

```diff
@@ -134,58 +134,58 @@ export const deTranslations: DefaultTranslationsObject = {
   },
   fields: {
     addLabel: '{{label}} hinzufügen',
-    addLink: 'Link Hinzufügen',
-    addNew: 'Neu erstellen',
+    addLink: 'Link hinzufügen',
+    addNew: 'Neuen Eintrag hinzufügen',
     addNewLabel: '{{label}} erstellen',
-    addRelationship: 'Verknüpfung Hinzufügen',
-    addUpload: 'Hochladen Hinzufügen',
+    addRelationship: 'Beziehung hinzufügen',
+    addUpload: 'Neue Datei hochladen',
     block: 'Block',
     blocks: 'Blöcke',
     blockType: 'Block-Typ',
     chooseBetweenCustomTextOrDocument:
-      'Wähle zwischen einer eigenen Text-URL oder verlinke zu einem anderen Dokument.',
-    chooseDocumentToLink: 'Wähle ein Dokument zum Verlinken',
+      'Wähle zwischen der Eingabe einer benutzerdefinierten URL oder verknüpfe ein anderes Dokument.',
+    chooseDocumentToLink: 'Wähle ein Dokument, das du verlinken möchtest',
     chooseFromExisting: 'Aus vorhandenen auswählen',
     chooseLabel: '{{label}} auswählen',
     collapseAll: 'Alle einklappen',
     customURL: 'Eigene URL',
     editLabelData: '{{label}} bearbeiten',
-    editLink: 'Bearbeite Link',
-    editRelationship: 'Beziehung Hinzufügen',
+    editLink: 'Link bearbeiten',
+    editRelationship: 'Beziehung hinzufügen',
     enterURL: 'URL eingeben',
-    internalLink: 'Interner Link',
+    internalLink: 'Interne Verlinkung',
     itemsAndMore: '{{items}} und {{count}} mehr',
-    labelRelationship: '{{label}} Verknüpfung',
+    labelRelationship: '{{label}}-Beziehung',
     latitude: 'Breitengrad',
     linkedTo: 'Verweist auf <0>{{label}}</0>',
     linkType: 'Linktyp',
     longitude: 'Längengrad',
     newLabel: '{{label}} erstellen',
-    openInNewTab: 'Öffne im neuen Tab',
+    openInNewTab: 'In neuem Tab öffnen',
     passwordsDoNotMatch: 'Passwörter stimmen nicht überein.',
     relatedDocument: 'Verknüpftes Dokument',
     relationTo: 'Verknüpfung zu',
-    removeRelationship: 'Beziehung Entfernen',
-    removeUpload: 'Hochgeladene Datei Löschen',
+    removeRelationship: 'Verknüpfung entfernen',
```

Comment:
I'm okay to use either "Verknüpfung" or "Beziehung" - can you update this to use either of those consistently? Then I think this is good to go